### DISCUSSION TITLE
[archived] "Scaling" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10266] [General]
# argo-workflows/scaling/

https://argoproj.github.io/argo-workflows/scaling/

**Comment by nicolas-vivot** [comment]
Could you please document what are the qps and burst arguments as well as how to use and configure it ?

**Reply by gyanprakash48** [reply]
@nicolas-vivot did you find its meaning and any example what value we can give , for large scaling

**Reply by nicolas-vivot** [reply]
qps seems to be the "query per seconds". It relates to how many pods may be created every seconds. I read somewhere (but cannot find the source) that it should have a value equals to the double of expected pod creation per second.

But yeah, not really sure how high that should be setup, especially how to configure the other configuration properties accordingly.
Would be great if someone with experience could give a few concrete example about how much vCPU/Memory to allocate + the configuration properties per scenario (100 regular concurrent multi steps workflows, 1000 concurrent steps, etc.)

**Reply by gyanprakash48** [reply]
Thanks for sharing these details.

**Reply by sgutwein** [reply]
Added a issue for that: https://github.com/argoproj/argo-workflows/issues/10566

**Reply by agilgur5** [reply]
#11731 recently resolved #10566 and added a solid amount to this page 🚀
